com.vrcfury.vrcfury/Editor/VF/Builder/MutableManager.cs

```diff
@@ -117,6 +117,8 @@ public class MutableManager {
                     copyClip.WriteProxyBinding(originalClip);
                 }
 
+                VRCFuryAssetDatabase.CopyAssetLabelsFromOriginal(original, copy);
```

Comment:
I'm curious how this interacts with sub-assets. Glancing at the way this is stored (in the top of the meta file), I'm afraid that setting labels on sub-assets will clobber the labels on the main asset in the file.

Reply:
Tested this on an FX Controller with 2 empty animation clips that have different labels: 'Vegatation' and 'Effect'. The final FX controller and all the sub-assets received the 'Effect' label as that was the last clip that was copied over. So yeah it's what you expected to happen _and_ we start to lose label information as well.

To fix the information loss, we could start to accumulate labels on the `saveParent`, but seems like if we do SetLabels on an object, then GetLabels on the object _immediately_ after, we don't get the same labels back. Calling `EditorUtility.SetDirty(copy); AssetDatabase.SaveAssets();` didn't seem to change anything about it either. So that'd require some extra book-keeping.

Though, is it necessary to keep track of labels on sub-assets in the first place? With how sub-assets clobber the labels of the main asset, labels seem to lose their usefulness from my perspective. I'm also assuming that sub-assets will mostly be used for animation controller related objects and I don't believe I've ever ran into those having labels applied to them.

If we don't want to keep this sub-asset label weirdness, we could move `VRCFuryAssetDatabase.CopyAssetLabelsFromOriginal` call into the `if (saveParent == null && saveFilename != null) {` branch instead.

Reply:
Yeah, seems like it's not super useful if it's only attached to the parent asset. We're actually planning on an upcoming change where no assets will be saved during the entire vrcfury bake process (because saving assets is SLOOOOOW), and then walking the avatar and saving everything out at the very end into like one big asset fail, which would make this not work :(. I'm open to other options.